phpBB/phpbb/db/migration/data/v32x/f_list_topics_permission_add.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v32x;
+
+class f_list_topics_permission_add extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array(
+			'\phpbb\db\migration\data\v32x\v321',
+		);
+	}
+
+	public function update_data()
+	{
+		return array(
+			array('permission.add', array('f_list_topics', false)),
```

Comment:
Actually, can you copy the permission from `f_read`? I think that might make sense.

Reply:
As in grant everyone with `f_read` the `f_list_topics` permission as well?

Reply:
Yep. Just so we have a standard value for existing installations that does make sense.